### PR TITLE
[FIX] account move line default_get

### PR DESCRIPTION
**Pull request by PieterPaulussen**
Description of the issue/feature this PR addresses:
The default get does not support multiple account_move_line's calculated debit/credit.

Current behavior before PR:
The move line's debit and credit are calculated from a hardcoded counterpart. (line[2]).

Desired behavior after PR is merged:
Return a calculated debit/credit from all move lines related to the move.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I don't get what is the problem, but you shouldn't change so much code if you want this to be clear for having it merged. You need to sign the CLA also.

**Comment by PieterPaulussen** [comment]
@pedrobaeza The diff is 10 lines. Are you seeing something else than me? 
I added the CLA thanks.

**Comment by PieterPaulussen** [comment]
@pedrobaeza Thanks for the tip! I simplified the PR.

**Comment by pedrobaeza** [comment]
Is this applicable also to other branches?

@qdp-odoo can you review this one?

**Comment by PieterPaulussen** [comment]
@pedrobaeza Well, V9.0 and V10.0 both didn't include a default get for account_move_line. So either it was solved in a different way, or it's isn't included at all. V8.0 has a completely different implementation (Which by the way is very slow. in computation.).

**Comment by pedrobaeza** [comment]
OK, thanks for the insight. Take into account that master is an alpha version, so some problems are expected to happen. Any way, it's good to have this fixed.

File: addons/account/models/account_move.py [hunk 1]
```diff
@@ -638,9 +638,8 @@ def default_get(self, fields):
 
         #compute the default credit/debit of the next line in case of a manual entry
         balance = 0
-        for line in self._context['line_ids']:
-            if line[2]:
-                balance += line[2].get('debit', 0) - line[2].get('credit', 0)
+        for line in self.env['account.move'].resolve_2many_commands('line_ids', self._context['line_ids']):
```
Comment (StefanRijnhart) [inline addons/account/models/account_move.py]: `resolve_2many_commands` is defined on the model level, not specifically on account.move so you can use `self.resolve_2many_commands`

Comment (StefanRijnhart) [reply addons/account/models/account_move.py]: You can pass an extra parameter with the fields you need. `fields=['debit', 'credit']`

Comment (StefanRijnhart) [reply addons/account/models/account_move.py]: I'm sorry, you are right. It's the line_ids field of account.move of course.

Comment (PieterPaulussen) [reply addons/account/models/account_move.py]: @StefanRijnhart I updated the PR and fixed the CLA to the corporate version in a separate commit.

Comment (StefanRijnhart) [reply addons/account/models/account_move.py]: PR is looking good! I think legal wise, Stijn Houben needs to commit the addition to the corporate CLS

Comment (PieterPaulussen) [reply addons/account/models/account_move.py]: Well, I discussed it internally with the Dynapps developers and they agreed I should do this: https://github.com/odoo/odoo/pull/25336. ;-)

**Comment by qdp-odoo** [comment]
nice catch. 

Landed in master, at revision https://github.com/odoo/odoo/commit/50ee1c964b5994c5e08eaf7f9ef800e0437f1672

Thank you all for the bug report, the fix and the reviews :-)

**Pull request closed without merging**